packages/react/test/profiler.test.tsx

```diff
@@ -1,6 +1,7 @@
 import type { SpanContext } from '@sentry/types';
 import { render } from '@testing-library/react';
 import { renderHook } from '@testing-library/react-hooks';
+// biome-ignore lint/nursery/noUnusedImports: Need React import for JSXq
```

Comment:
```suggestion
// biome-ignore lint/nursery/noUnusedImports: Need React import for JSX
```

Reply:
Done in 9249efed4f33be0105491b8f7721e21d8c3ba47b